### DISCUSSION TITLE
CI-Examples/python

### DISCUSSION THREAD

**Discussion by reclock** [gramineproject/gramine#1659] [Q&A] [answered]
I executed sgx report. py and the test was successful, but the execution of sgx quote. py failed with the following message:

Cannot find `/dev/attestation/quote `; Are you running under SGX with remote testing enabled?

Why is this? Unable to read quote information

**Comment by kailun-qin** [comment, marked as answer]
@reclock: Were you trying to run our [CI-Examples/python](https://github.com/gramineproject/gramine/tree/master/CI-Examples/python) and following the README instructions there? Would you pls enable debug and share the Gramine logs w/ us? Thanks!

**Comment by reclock** [comment]
Sorry, I forgot to include during compilation RA_TYPE=dcap
Now it's ok
